pkg/utils/setuputils/setup_syncer.go

```diff
@@ -67,12 +66,10 @@ func (s *SetupSyncer) Sync(ctx context.Context, snap *v1.SetupSnapshot) error {
 	watchedNamespaces := settingsutil.GetNamespacesToWatch(settings)
 	contextutils.LoggerFrom(ctx).Debugw("received updated list of namespaces to watch", zap.Any("namespaces", watchedNamespaces))
 
-	watchedNamespacesStr := strings.Join(watchedNamespaces, ",")
 	statsutils.Measure(
 		ctx,
 		mNamespacesWatched,
 		int64(len(watchedNamespaces)),
-		tag.Insert(namespacesWatchedKey, watchedNamespacesStr),
```

Comment:
This was not used and caused an error when the concatenated list of namespaces > 255 chars